aws/resource_aws_glue_crawler_test.go

```diff
@@ -893,26 +1018,26 @@ data "aws_partition" "current" {}
 
 resource "aws_iam_role" "test" {
   name = %q
+  assume_role_policy = "${data.aws_iam_policy_document.assume.json}"
+}
 
-  assume_role_policy = <<EOF
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Action": "sts:AssumeRole",
-      "Principal": {
-        "Service": "glue.amazonaws.com"
-      },
-      "Effect": "Allow",
-      "Sid": ""
+data "aws_iam_policy_document" "assume" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["glue.amazonaws.com"]
     }
-  ]
+  }
 }
-EOF
+
+data "aws_iam_policy" "AWSGlueServiceRole" {
+  arn = "arn:aws:iam::aws:policy/service-role/AWSGlueServiceRole"
```

Comment:
Please keep the usage of the `aws_partition` data source -- we run this acceptance testing across multiple AWS partitions like GovCloud (US).

```suggestion
  arn = "arn:${data.aws_partition.current.partition}:iam::aws:policy/service-role/AWSGlueServiceRole"
```